main_objs.py

```diff
@@ -480,7 +501,7 @@ def _cull_ngram_helper(self, ser):
 
     def getNgrams(self, df=None, n=3, how='columnwise', other=None, held='Held',
                   exclude=['Rest'], interval_settings=('d', True, True),
-                  cell_type=tuple, unit=0, max_n=0, report=False):
+                  cell_type=tuple, unit=0, max_n=0, offsets='start', report=False):
```

Comment:
max_n and report go together as settings, so it would be nice to leave them together. Why don't your put the offsets setting right before max_n? Also, you need to add a little bit about the offsets setting to the doc string for getNgrams since it's a public method. The best place would be right after the unit=2 example. Also you really need to explain that "end" means the last onset, because a lot of people might think that it's the end of the last event, but it's the beginning of the last event.